README.md

```diff
@@ -4,47 +4,86 @@
 Placeholder for the Metadata Query Service
 
 
+# Query Service
+[![Test Coverage](https://codecov.io/gh/HumanCellAtlas/query-service/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/query-service)
+[![Master Deployment Status](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/badges/master/pipeline.svg)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/commits/master)
```

Comment:
```suggestion
[![Master Build Status](https://status.dev.data.humancellatlas.org/build/HumanCellAtlas/metrics/master.svg)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/commits/master)
```

Reply:
Unfortunately this won't work until this bug is fixed: https://gitlab.com/gitlab-org/gitlab-ce/issues/54633

Reply:
kk